docs/security/users-and-teams/auditing.md

```diff
@@ -45,14 +50,37 @@ We take great care to ensure the security and integrity of your audit logs, to m
 
 To grant a user access to audit logs you can make use of a built-in User Role that contains **EventView**. All project related user roles contain it. **EventView** can also be scoped to narrow down which audit information a user can see, for example, it can be restricted to specific Projects or Environments. Learn more about [managing users and teams](/docs/security/users-and-teams/index.md).
 
+Audit entries older than the configured retention period (defaults to 90 days, configurable up to 365 days) are archived and can be accessed via the overflow menu (`...`) in the top right corner of the audit page by selecting the **Manage archived events** option.
+
 In **Octopus 2019.1** we removed **AuditView** in an effort to simplify permissions so only **EventView** is now required.
 
 ### Modifying and deleting audit logs is prevented
 
-Octopus actively prevents modifying or deleting audit logs via its API. That being said, a user with the appropriate permissions to the `Events` table in your Octopus SQL Database could modify or delete records in that table. If you are concerned about this kind of tampering you should configure the permissions to the `Events` table in your Octopus SQL Database appropriately.
+Octopus actively prevents modifying or deleting audit logs within the configured retention period via its API. That being said, a user with the appropriate permissions to the `Events` table in your Octopus SQL Database could modify or delete records in that table. If you are concerned about this kind of tampering you should configure the permissions to the `Events` table in your Octopus SQL Database appropriately.
+
+Entries older than the retention period can be deleted by users with the appropriate permissions (typically `Octopus Manager`). An audit log entry will be created each time an archived event file is deleted. Archived files are saved at a filesystem level. So any user that has the appropriate permissions could view or delete these files. If this is a concern, you should restrict the permissions to access the configured folder appropriately.
 
 ### Sensitive values in audit logs
 
 If you make a change to a sensitive value in Octopus, you will notice we write an audit log showing the fact the sensitive value changed. The value we show in the audit log is simply **an indicator the value has changed**. This is **not** the unencrypted/raw value. This is **not** even the encrypted value.
 
 We take the sensitive value and hash it using an irreversible hash algorithm. We then encrypt that hash with a new, unique, non-deterministic salt. We use this irreversible value as an indicator that the sensitive value actually changed in some way.
+
+### Manage archived audit logs {#manage-archived-audit-events}
+
+:::hint
+Audit Retention functionality was introduced in **Octopus 2022.3** and is being rolled out in stages for our Cloud customers first. It will be made available to Self-hosted customers at a later stage.
```

Comment:
```suggestion
Audit Retention functionality was introduced in **Octopus 2022.4** and is being rolled out in stages for our Cloud customers first. It will be made available to Self-hosted customers at a later stage.
```

Reply:
According to the latest information I saw, this would now be introduced in 2022.4

Reply:
We're rolling out to cloud customers in 2022.3 but it will be available for on-prem in 2022.4. Does that mean this should be 2022.4 or should we change the wording?

Reply:
We could opt to change the wording to:

```suggestion
Audit Retention functionality was introduced in **Octopus 2022.3** and is being rolled out in stages for our Cloud customers first. It will be made available to Self-hosted customers in **Octopus 2022.4**.
```

But the wording is accurate, it might be confusing if anyone wonders what "later stage" means.